client/src/components/atoms/SubmissionToolbar.vue

```diff
@@ -2,29 +2,27 @@
   <q-header reveal class="bg-grey-9 text-white">
     <q-toolbar>
       <q-btn
+        :aria-label="$t('submission.toolbar.back')"
         dense
-        aria-label="Back to Submission Details"
         flat
         round
         icon="arrow_back_ios_new"
-        :to="{
-          name: 'submission_details',
-          params: { id: props.id },
-        }"
+        :to="{ name: 'submission_details', params: { id: props.id } }"
       />
       <q-toolbar-title>
         {{ submission.title }}
       </q-toolbar-title>
 
       <q-btn
-        aria-label="Toggle Annotations"
+        :aria-label="$t('submission.toolbar.toggle_annotation_highlights')"
         dense
         flat
         round
         icon="power_input"
+        @click="toggleAnnotationHighlights"
       />
```

Comment:
This needs a tooltip or something.  Its not clear at all what this button does unless one is using a screen reader.

Reply:
@wreality Changed as suggested. 

I also simplified the label "Toggle Annotation Highlights" to "Toggle Highlights" since the label 'annotations' in this context is more useful for developers and less so for users.